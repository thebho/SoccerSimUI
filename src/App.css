.App {
  text-align: left;
}

.App-header {
  background-color: #222;
  padding: 10px;
  padding-left: 30px;
  color: white;
}

.App-title {
  font-size: 30px;
}

.App-intro {
  font-size: large;
}
