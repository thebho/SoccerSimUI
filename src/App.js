// @flow
import React, { Component } from 'react';
import './App.css';
import TeamsTable from './components/TeamsTable';
import MatchWeek from './components/MatchWeek';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';


class App extends Component<*> {
  render() {
    return (
      <MuiThemeProvider>
        <div className='App'>
          <header className='App-header'>
            <h1 className='App-title'>Mock English Football League</h1>
          </header>
          <div className='App-body'>
            <TeamsTable className='App-teams'/>
            <MatchWeek className='App-schedule' store={this.props.store}/>
          </div>
        </div>
      </MuiThemeProvider>
    );
  }
}

export default App;
