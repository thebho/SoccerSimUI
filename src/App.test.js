import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('App.js', () => {
  it('matches snapshot with defaults', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
