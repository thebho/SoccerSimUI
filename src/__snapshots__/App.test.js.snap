// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.js matches snapshot with defaults 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Mock English Football League
    </h1>
  </header>
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
